src/evidence.cpp

```diff
@@ -413,6 +413,15 @@ void Courtroom::on_evidence_image_button_clicked()
     return;
 
   QString filename = filenames.at(0);
+  QStringList bases = ao_app->get_mount_paths();
+  bases.prepend(ao_app->get_base_path());
+  for (const QString &base : bases) {
+    QDir baseDir(base);
+    if (filename.startsWith(baseDir.absolutePath())) {
+      dir.setPath(baseDir.absolutePath() + "/evidence");
+      break;
+    }
+  }
```

Comment:
This belong with the path functions in my opinion. Too bad I didn't implement a generalized way to map from filesystem to vpath.